src/internal/fungible_asset.ts

```diff
@@ -54,3 +59,47 @@ export async function getFungibleAssetMetadata(args: {
 
   return data.fungible_asset_metadata;
 }
+
+export async function getFungibleAssetActivities(args: {
+  aptosConfig: AptosConfig;
+  options?: {
+    pagination?: PaginationArgs;
+    whereCondition?: {
+      ownerAddress?: HexInput;
+      assetType?: string;
+      type?: string;
+    };
+  };
+}): Promise<GetFungibleAssetActivitiesResponse> {
+  const { aptosConfig, options } = args;
+
+  const whereCondition: any = {};
+  if (options?.whereCondition?.ownerAddress) {
+    whereCondition.owner_address = {
+      _eq: AccountAddress.fromHexInput(options.whereCondition.ownerAddress).toString(),
```

Comment:
one of the reasons being that options.whereCondition.ownerAddress is a bit clunky.

set it to a variable?